docs/src/main/tut/entity.md

```diff
@@ -96,7 +96,7 @@ If you're working with [twirl] templates, there's a bridge for that too:
 
 - scala-twirl: `"org.http4s" %% "http4s-twirl" % Http4sVersion`
 
-[streams-tutorial]: https://gist.github.com/djspiewak/d93a9c4983f63721c41c
+[introduction-to-functional-streams]: https://github.com/functional-streams-for-scala/fs2
```

Comment:
This PR will be perfect with this line changed to 

```
[introduction-to-functional-streams]: https://youtu.be/cahvyadYfX8
```

Sorry for all the tiny changes!

Reply:
@ChristopherDavenport No problem!! Should be fixed now.